ui_framework/doc_site/src/views/expression/expression.html

```diff
@@ -0,0 +1,32 @@
+<div class="kuiExpression">
+  <div class="kuiExpression__item">
+    <button data-id="toggleExpressionButton" class="kuiExpression__button">
+      <span class="kuiExpression__buttonDescription">WHEN</span>
+      <span class="kuiExpression_buttonValue">count()</span>
+    </button>
+    <div  data-id="toggleExpressionPopover" class="kuiExpression__popover kuiExpression__popover--hidden">
```

Comment:
Looks like an extra space after `<div` here. And can you split the attributes across multiple lines?